frontend/src/components/video/ViewVideo.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from "react"
```

Comment:
This is basically a replica of VideoViewPage but with more props for easier navigation and selection.